mcrouter/lib/network/gen/MemcacheRouterStats.h

```diff
@@ -25,33 +25,37 @@ namespace facebook {
 namespace memcache {
 
 struct MemcacheRouterStatsConfig {
-  static constexpr size_t kNumRequestGroups = 6;
+  static constexpr size_t kNumRequestGroups = 7;
   static constexpr std::array<folly::StringPiece, 1 * kNumRequestGroups>
       sumStatNames{{folly::StringPiece("cmd_delete_count"),
                     folly::StringPiece("cmd_get_count"),
                     folly::StringPiece("cmd_lease_get_count"),
                     folly::StringPiece("cmd_lease_set_count"),
                     folly::StringPiece("cmd_set_count"),
-                    folly::StringPiece("cmd_other_count")}};
+                    folly::StringPiece("cmd_other_count"),
+                    folly::StringPiece("cmd_gets_count")}};
   static constexpr std::array<folly::StringPiece, 3 * kNumRequestGroups>
       rateStatNames{{folly::StringPiece("cmd_delete"),
                      folly::StringPiece("cmd_get"),
                      folly::StringPiece("cmd_lease_get"),
                      folly::StringPiece("cmd_lease_set"),
                      folly::StringPiece("cmd_set"),
                      folly::StringPiece("cmd_other"),
+                     folly::StringPiece("cmd_gets"),
```

Comment:
`rateStatNames` is also order dependent. It must be exactly 3 times the length of `sumStatNames`. The ordering must match the same as the count metrics too. 

order will be:

1. all the `cmd_N` rate metrics in the same order of count metrics
2. all the `cmd_N_out` rate metrics in same order of count metrics
3. all the `cmd_N_all` rate metrics in same order of count metrics